gensim/models/word2vec.py

```diff
@@ -200,6 +200,11 @@
 from gensim.models.keyedvectors import KeyedVectors, pseudorandom_weak_vector
 from gensim import utils, matutils
 
+#
+# This import is required by pickle to load models stored by Gensim < 4.0, such as Gensim 3.8.3.
+#
```

Comment:
No need for this blank line – the comment above relates directly to the line below.